src/tests/example.spec.js

```diff
@@ -24,4 +31,131 @@ test.describe('', () => {
         await shopingCartPage.removeCartItemById(0);
         await expect(shopingCartPage.cartItems).not.toBeAttached();
     });
+
+    const sortingDirections = ['az', 'za', 'lohi', 'hilo'];
+
+    for (const direction of sortingDirections){
+        if (direction === 'az'){
+            test('Verify sorting by name in ASC', async ({ inventoryPage }) => {
```

Comment:
The main point here is to make test code reusable for each of sortingDirections. This should looks like this: 
``` for (const direction of sortingDirections){
            test(`Verify sorting by name in ${direction}`, async ({ inventoryPage }) => {
                ...
            })
    })
```

Reply:
@hlazkov yea, i agree that  using if operator doesn't make sense here, though if i proceed without it, test fails since i'm trying to execute all for tests for one specific direction. Unfortunately, I don't have solution how to deal with it/ Maybe you can help with this?